models/intel/icnet-camvid-fp32-onnx-0001/description/icnet-camvid-fp32-onnx-0001.md

```diff
@@ -0,0 +1,56 @@
+# icnet-camvid-fp32-onnx-0001
+
+## Use Case and High-Level Description
+
+This is the ICNet model that is designed to perform semantic segmentation. The model has been trained on the CamVid dataset from scratch using PyTorch framework. Training used median frequency balancing for class weighing and random horizontal flips as data augmentation. For details about the original floating point model, check out the [paper](https://arxiv.org/pdf/1704.08545.pdf).
+
+The model input is a blob that consists of a single image of "1x3x768x768" in BGR order. The pixel values are floating-point values that were obtained by taking each channel value as an integer in the [0, 255] range and dividing it by 255.
```

Comment:
and division by 255 also should be removed, IR expects floating-point values in the range [0, 255]